src/app/feed/feed-create-post/feed-create-post.component.html

```diff
@@ -39,7 +39,7 @@
             rows="{{ parentPost ? 3 : numberOfRowsInTextArea }}"
             class="form-control fs-18px m-5px p-0 border-0 feed-create-post__textarea position-relative background-color-white"
             id="textarea-{{ i + 1 }}"
-            [placeholder]="t(postModels[i].placeHolderText)"
+            [placeholder]="postModels[i].showPoll ? 'Ask Question' : t(postModels[i].placeHolderText)"
```

Comment:
should it be `Ask a Question`?

Reply:
Thanks for the heads up - updated the copy